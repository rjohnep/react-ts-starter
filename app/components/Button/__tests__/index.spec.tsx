import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { Button } from '../index';

const renderer = ShallowRenderer.createRenderer();

describe('<Button />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<Button />);

    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
