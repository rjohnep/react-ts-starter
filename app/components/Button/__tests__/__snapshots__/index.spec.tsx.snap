// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render and match the snapshot 1`] = `
<styled.div
  variant="Primary"
/>
`;
